application/settings.py

```diff
@@ -37,15 +37,15 @@
     'django.contrib.contenttypes',
     'django.contrib.staticfiles',
     'django.contrib.gis',
+    'django.contrib.auth',
```

Comment:
Otherwise with new Django I have 

```
Model class django.contrib.auth.models.Permission doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.
```

error